app/editor/src/features/content/tool-bar/sections/filter/AdvancedSearchSection.tsx

```diff
@@ -86,7 +89,36 @@ export const AdvancedSearchSection: React.FC<IAdvancedSearchSectionProps> = () =
               )}
             />
           </Show>
-          <Show visible={filterAdvanced.fieldType !== 'sourceIds'}>
+          <Show visible={filterAdvanced.fieldType === advancedSearchKeys.Status}>
```

Comment:
We now have better UI for status options.  I need to do the dates at some point.